### ISSUE TITLE
"Could not find minikube binary" error 

### ISSUE THREAD

**Issue by bitfield** [Azure/vscode-kubernetes-tools#443]
I don't use Minikube, but the Kubernetes extension is complaining that it's not there nevertheless:

![screen shot 2019-01-22 at 16 28 48](https://user-images.githubusercontent.com/142262/51549849-0c643480-1e63-11e9-8639-50d9e62dc965.png)

I'm guessing this is related to #300, but the fact is, I don't _want_ to install Minikube, and I'm not doing anything that requires it. I don't think I should be seeing this error dialog unless I do something in VS Code that requires Minikube, should I?

**Comment by itowlson** [comment]
Agree, this is an annoyance; I think we are trying to check if Minikube is up to date and are not doing the 'find Minikube to run this check' bit silently.  We should prompt only if there is a Minikube cluster in kubeconfig, or indeed only if Minikube is activated.  Thanks for flagging this!
